loadgen/loadgen.cc

```diff
@@ -90,7 +99,7 @@ class QueryMetadata {
         wait_count_(query_sample_indices.size()) {
     samples_.reserve(query_sample_indices.size());
     for (QuerySampleIndex qsi : query_sample_indices) {
-      samples_.push_back({this, sequence_gen->NextSampleId(), qsi});
+      samples_.push_back({this, sequence_gen->NextSampleId(), qsi, sequence_gen->NextAccLogRng()});
```

Comment:
Hm did you want this renamed to NextAccuracyLogRandomNumber or something?

Reply:
Don't worry about it. Following to the body of NextAccLogRng makes things clear if it wasn't already.

Reply:
This line looks like it exceeds 80 characters. clang-format?

Reply:
Thanks for noticing this. Not sure why clang-format didn't catch it the first time.